src/Compilers/CSharp/Portable/Symbols/SymbolWithAnnotations.cs

```diff
@@ -208,10 +208,9 @@ public TypeSymbolWithAnnotations AsNullableReferenceOrValueType(CSharpCompilatio
         public abstract bool IsAnnotated { get; }
 
         /// <summary>
-        /// [NonNullTypes] context used for determining
-        /// whether unannotated types are not nullable.
+        /// Lazy [NonNullTypes] context used for determining whether unannotated types are not nullable.
```

Comment:
>Lazy [](start = 12, length = 4)

Minor point: Consider removing `Lazy` since the context itself is not lazy, just the calls to the `NonNullTypes` property are.